libexec/scoop-list.ps1

```diff
@@ -28,38 +28,38 @@ Write-Host "Installed apps$(if($query) { `" matching '$query'`"}):"
 $apps | Where-Object { !$query -or ($_.name -match $query) } | ForEach-Object {
     $app = $_.name
     $global = $_.global
-    $item = @{ PSTypeName = 'ScoopAppsList' }
+    $item = @{}
     $ver = Select-CurrentVersion -AppName $app -Global:$global
     $item.Name = $app
     $item.Version = $ver
 
     $install_info_path = "$(versiondir $app $ver $global)\install.json"
-    $updated = (Get-Item (appdir $app $global)).LastWriteTime | Get-Date -Format 'yyyy-MM-dd HH:mm:ss'
+    $updated = (Get-Item (appdir $app $global)).LastWriteTime
     $install_info = $null
-    if(Test-Path $install_info_path) {
+    if (Test-Path $install_info_path) {
         $install_info = parse_json $install_info_path
-        $updated = (Get-Item $install_info_path).LastWriteTime | Get-Date -Format 'yyyy-MM-dd HH:mm:ss'
+        $updated = (Get-Item $install_info_path).LastWriteTime
     }
 
     $item.Source = if ($install_info.bucket) {
         $install_info.bucket
     } elseif ($install_info.url) {
-        if ($install_info.url -eq (usermanifest $app)) { "<auto-generated>" }
+        if ($install_info.url -eq (usermanifest $app)) { '<auto-generated>' }
         else { $install_info.url }
     }
     $item.Updated = $updated
 
     $info = @()
-    if($global) { $info += "Global install" }
-    if (failed $app $global) { $info += "Install failed" }
-    if ($install_info.hold) { $info += "Held package" }
+    if ($global) { $info += 'Global install' }
+    if (failed $app $global) { $info += 'Install failed' }
+    if ($install_info.hold) { $info += 'Held package' }
     if ($install_info.architecture -and $def_arch -ne $install_info.architecture) {
         $info += $install_info.architecture
     }
     $item.Info = $info -join ', '
 
-    $list += $item
+    $list += [PSCustomObject]$item
 }
 
-$list.ForEach({[PSCustomObject]$_})
+$list | Add-Member -TypeName 'ScoopApps' -PassThru
```

Comment:
Won't this create a new object every time (inefficient)? What's the advantage of doing this vs. specifying the type name on Line 31?

Reply:
No, this just `Add-Member`, i.e., affects only `$list`.

This is more clear, and create normal PSObject unless we need a Table View for the array of ScoopApp item, i.e., we just modify the final output.